src/api/resources/me.py

```diff
@@ -0,0 +1,35 @@
+from rest_framework import mixins
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework import serializers
+from rest_framework.viewsets import GenericViewSet
+
+from .base import BaseAPISerializer
+from ..models import Profile
+
+
+class ProfileSerializer(BaseAPISerializer):
+    countries = serializers.SerializerMethodField()
```

Comment:
Maybe should use module's way of doing DRF integration?:
https://github.com/SmileyChris/django-countries#django-rest-framework
In `StatsSerializer` I did `country = CountryField(country_dict=True)` but of course that's just for an instance

Reply:
I may be mistaken but it looks like I would need to create a serializer to do that

Reply:
Note for future selves: Using `CountryFieldMixin` worked except it just output a list of code strings. In theory one should be able to combine this with the `CountryField(country_dict=True)` to yield a list of dicts, but as far as I can tell the two don't play nicely together. So, fall back on tried and true.